bot/src/wrapper/_communication.py

```diff
@@ -4,9 +4,9 @@
 
 import aiohttp
 
-from ._api_schema import (  # Company
+from ._api_schema import (
     AchievementGetOutput,  # Achievement
-    AchievementIdGetOutput,
+    AchievementIdGetOutput,  # Company
```

Comment:
The comment should be one line lower for `# Company` if it is not the first import